SECURITY.md

```diff
@@ -0,0 +1,7 @@
+# Security Policy
+
+The project supports the current version of the assets available at https://dpel.org
+
+## Reporting a Vulnerability
+
+If you would like to report a security vulerability, you can open a security report at https://github.com/AcademySoftwareFoundation/dpel-website/security/advisories/new, or send to security@dpel.org.
```

Comment:
Where does security@dpel.org go?  Is that an alias for assets-submission@lists.aswf.io?

Reply:
Yep - that is correct.